DataStructuresMemories/DataStructProtocol.swift

```diff
@@ -8,7 +8,8 @@
 
 import Foundation
 
-protocol DataStruct {
+protocol DataStructProtocol {
```

Comment:
why only `get` and not `get set`?

Reply:
I've added only 'get', because it  and it sets in constructor. Should I add 'get'?

Reply:
...also I wanted to hide the rewrite of that properies